rpc/src/lib.rs

```diff
@@ -12,17 +12,20 @@ extern crate tonic;
 #[allow(clippy::upper_case_acronyms)]
 #[allow(clippy::derive_partial_eq_without_eq)]
 pub mod io_engine {
-    use bdev_rpc_client::BdevRpcClient;
-    /// AutoGenerated Io Engine Client
-    pub use mayastor_client::MayastorClient as IoEngineClient;
+    /// AutoGenerated Io Engine V1 Nexus Client.
+    use super::v1::nexus::nexus_rpc_client::NexusRpcClient as IoEngineClientV1;
```

Comment:
Why only nexus client is treated as `IoEngineClientV1`? Would it not be better if we add all the clients for v1 in a wrapper and them that as `IoEngineClientV1`?

Reply:
This is just used by tests and only the nexus was needed, yeah I can do that.

Reply:
Done.